generators/app/templates/stories/components.js

```diff
@@ -103,3 +104,54 @@ storiesOf('Checkbox', module).add('With label', () => <Checkbox label="Acepto t
 storiesOf('Checkbox', module).add('With custom style', () => <Checkbox inputClassName={styles.checkmark} />);
 
 storiesOf('Checkbox', module).add('With callback', () => <Checkbox onChange={action('clicked')} />);
+
+storiesOf('RadioGroup', module).add('Default', () => (
+  <RadioGroup>
+    <RadioGroup.option label="option 1" value="1" />
+    <RadioGroup.option label="option 2" value="2" />
+    <RadioGroup.option label="option 3" value="3" />
+  </RadioGroup>
+));
+
+storiesOf('RadioGroup', module).add('With custom styles', () => (
+  <RadioGroup className={styles.radioGroup}>
+    <RadioGroup.option
```

Comment:
I think this might not be the best application for this.
Why not something like 
```js
import { RadioGroup, RadioOption } from '../src/app/components/RadioGroup';
...
```

and in the RadioGroup.js
```js 
export class RadioGroup {
...
}

export const RadioOption = RadioGroup.option;
```

Reply:
I'd let this as is, since it's magic :sparkles:
Notice that this structure lets `RadioGroup` to receive a prop and share it with specific children without cloning them. By example, by passing a `onChange` prop to `RadioGroup`, all `RadioGroup.option` childrens are aware of it without passing it explicitly 🌈

Reply:
More important, this approach doesn't break encapsulation